metrics/prometheus/prometheus.go

```diff
@@ -73,6 +73,8 @@ func (g gatherer) Gather() ([]*dto.MetricFamily, error) {
 					},
 				}},
 			})
+		case metrics.GaugeInfo:
+			// XXX
```

Comment:
not sure what to put here, not sure we need this either, see the comment for core/blockchain.go.